### PR TITLE
London Class 7 - Negin Derakhshan - HTMl/CSS - Week 1

### PR DESCRIPTION
**Pull request by NeginDn**
# Your Details

- Your Name: Negin Derakhshan
- Your City: London
- Your Slack Name: Negin DN

# Homework Details

- Module: Git / GitHub
- Week: 1

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,110 +4,102 @@
...
 
-Send you code to Github
+The "commit" command is used to save your changes to the local repository.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio
```
Comment (soraiya) [inline homework.md:97]: Visual Studio Code would be more accurate as the program name

File: homework.md [hunk 2]
```diff
@@ -4,110 +4,102 @@
...
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
-
-}
-```
+h1.intro
```
Comment (soraiya) [inline homework.md:37]: Double check answer

**Review by soraiya — COMMENTED** [review]
Answered very well, overall. A couple of minor changes proposed to Q6 & Q16 :)

File: homework.md [hunk 3]
```diff
@@ -4,110 +4,102 @@
...
 
 What happens when you `fork` a repository?
 
-You delete it
+You make a editable copy of someone else's for yourself
 
-## Question 10 
+## Question 10
 
 What happens when you clone a repostory?
```
Comment (DanGhanbari) [inline homework.md:60]: Hi Negin, there is a typo in question 10. 
Other than that you did a great job. well done

File: homework.md [hunk 4]
```diff
@@ -4,110 +4,107 @@
...
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```
Comment (louisechow) [inline homework.md:18]: It looks like you may have accidentally made change to the question here.

File: homework.md [hunk 5]
```diff
@@ -4,110 +4,107 @@
...
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS is the language for describing the presentation of Web pages, including colors, layout, and fonts.
```
Comment (louisechow) [inline homework.md:32]: I really like this answer! Especially how you pointed out that CSS is a language and that it describes presentation 🌟

File: homework.md [hunk 6]
```diff
@@ -4,110 +4,107 @@
...
 
-A programming language used on the internet
+JavaScript is a text-based programming language used both on the client-side and server-side
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+GitHub is a Git repository hosting service, but it adds many of its own features. While Git is a command line tool, GitHub provides a Web-based graphical interface. It also provides access control and several collaboration features, such as a wikis and basic task management tools for every project
```
Comment (louisechow) [inline homework.md:50]: This answer describes what features Git and Github have. But what problems do those features solve?
E.g. What problem does access control solve? What problem does a web-based graphical interface solve?

File: homework.md [hunk 7]
```diff
@@ -4,110 +4,107 @@
...
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You make a editable copy of someone else's for yourself
```
Comment (louisechow) [inline homework.md:56]: Where is this editable copy?

File: homework.md [hunk 8]
```diff
@@ -4,110 +4,107 @@
...
 
-It send it to a friend
+It downloads the repostory from web to your computer
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+Proposing your changes and requesting that someone review and pull in your contribution and merge them into their branch
```
Comment (louisechow) [inline homework.md:69]: The first part of this answer is perfect, but the second part is not always the case.

Sometimes you will be the person who owns the repository and you will send a pull request to other people to ask for a review. So, the other person will not be responsible for the merge.

File: homework.md [hunk 9]
```diff
@@ -4,110 +4,107 @@
...
 
-It lets you see into the future
+it log lists the commits made in that repository in reverse chronological order
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+When you push changes, you send the committed changes in your local repository to the remote repository on GitHub. If you change your project locally and want other people to have access to the changes, you must push the changes to GitHub.
```
Comment (louisechow) [inline homework.md:81]: 👍

File: homework.md [hunk 10]
```diff
@@ -4,110 +4,107 @@
...
 
-Microsoft Word
+Visual Studio Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+A commit message is attached to the change — not the code itself. When you write a commit message you are writing it as if it's about to be applied, rather than about what you just did.
```
Comment (louisechow) [inline homework.md:104]: Yes, this is considered a good way to write commit messages.

**Comment by louisechow** [comment]
The title of your pull request says HTML/CSS week 1 🤔. It would be good for you to change this, otherwise it might be confusing.

Otherwise, great job completing the homework 🌟

**Pull request closed without merging**